src/main/java/appeng/bootstrap/BlockDefinitionBuilder.java

```diff
@@ -268,7 +271,7 @@ private void customizeForClient( BlockRenderingCustomizer callback )
 
 		if( block instanceof AEBaseTileBlock )
 		{
-			this.factory.addPreInit( side ->
+			this.factory.<IPreInitComponent>addBootstrapComponent( side ->
 			{
 				AEBaseTile.registerTileItem(
 						this.tileEntityDefinition == null ? ( (AEBaseTileBlock) block ).getTileEntityClass() : this.tileEntityDefinition.getTileEntityClass(),
```

Comment:
Unrelated, but it might be an option and consider to remove the `getTileEntityClass` in favor of just `tileEntityDefinition` to handle everything with an explicit definition and not some implicit resolving through the `AEBaseTileBlock`. They might then need a way to initialise them with correct tilenentity class through the registration process instead of hardcoding the actual class into the block.